package.json

```diff
@@ -39,6 +39,7 @@
   },
   "homepage": "https://github.com/moshensky/pdf-visual-diff#readme",
   "dependencies": {
+    "@playwright/test": "^1.40.1",
```

Comment:
👋 

Thank you for your contribution. 
Are there any options not to depend directly on playwrigit?

Reply:
Ow, this one should be in the dev dependencies